db/comdb2.h

```diff
@@ -1320,11 +1320,7 @@ struct ireq {
     /* copy of blkseq */
     uint8_t seq[MAX_SEQ_LEN];
 
-    /* for blocksql purposely we want to have the list of sql
-       queries submitted;
-       the field is not null if we have any BLOCK2_SQL requests
-       THIS IS USED IN BLOCKSQL, WOULD BE NICE TO UNIFY -later-
-     */
+    /* bplog pointer */
     void *blocksql_tran;
```

Comment:
can we rename this for clarity?

Reply:
there is a bigger rework of this coming up